lib/user.js

```diff
@@ -105,5 +105,28 @@ module.exports = {
   },
   denyTracking: function(){
     refs.userMeta.child('trackOk').set( false );
+  },
+
+  /*
+   * Group Support
+   */
+  setGroup: function(key, devices) {
+    if (devices === null) { // Just create group without any device.
+      refs.userGroups.child(key).set(false);
+    }
+    refs.userGroups.child(key).set(devices);
+  },
+  deleteGroup: function(key) {
+    refs.userGroups.child(key).set(null);
```

Comment:
why not `remove()` ?

Reply:
removeGroup()? Just remove() isn't semantic because it'll be inside `matrix-firebase.user.[remove]()`